doc/specification.md

```diff
@@ -75,3 +77,120 @@ To enable interchangeability the files are characterized by
 
 The JSON files additionally follow
 * 2 spaces indentation
+
+## Type Specification
+
+Each JSON schema provided in this repository is automatically genereated. For this purpose, an interface corresponds
+to each ABAP object type in which the necessary components of the type are described in the type `ty_main`.
+One component of the type is the field `schema` which is translated to `$schema` in the JSON schema. Its value in the `.json` files is the link given in the field `$id` of the JSON schema. 
+
+In order to specify the type, ABAP Doc can be used. The comments areplaced directly above the components of the type `ty_main`, but they are also read over several levels as, e.g., in the case of nested structures. The different possibilities are summarized in the following.
+
+### Title
+To provide a title, the annotation
+```abap
+"! <p class="shorttext">Insert Title</p> 
+```
+is used. If no title is specified, the component name transformed to camel case is written. 
+
+### Description
+An ABAP Doc comment without annotations is passed as description to the JSON schema. 
+```abap
+"! This is the description
+```
+If no description is given, either the title if it is provided or the componenet name tranformed to camel case is shown in the JSON schema. 
+
+### Enum Values
+To pass enum values to a JSON schema, a type and a constant are specified. The constant consists of the internel (ABAP) values, the names of its components are written as external (JSON) values. Descriptions of the enum values are passed in the same way as above. 
+The type specifies the underlying data type and links to the constant via the following ABAP Doc annotation.
```

Comment:
I wouldn't call it ABAP doc annotation, since it is no official ABAP doc feature
```suggestion
The type specifies the underlying data type and links to the constant via the following annotation.
```